tiledb/sm/fragment/fragment_metadata.cc

```diff
@@ -3296,6 +3481,49 @@ Status FragmentMetadata::load_fragment_min_max_sum_null_count(
   return Status::Ok();
 }
 
+// ===== FORMAT =====
+// condition_num (uint64_t)
+// processed_condition_size#0 (uint64_t)
+// processed_condition#0
```

Comment:
Would be helpful t clarify what this value is here as well, for future readers.

Reply:
Added to the function declaration in the header.